modules/oe_whitelabel_search/oe_whitelabel_search.module

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @file
+ * OE Whitelabel Search Module.
+ */
+
+declare(strict_types =  1);
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter() for facets_forms.
+ */
+function oe_whitelabel_form_facets_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (isset($form['actions'])) {
+    $action_classes = ['mt-4', 'd-grid', 'gap-4', 'd-md-block'];
+    foreach ($action_classes as $action_class) {
+      if (!isset($action_class, $form['actions']['#attributes']['class']) || !in_array($action_class, $form['actions']['#attributes']['class'], TRUE)) {
+        $form['actions']['#attributes']['class'][] = $action_class;
+      }
+    }
+  }
+  if (isset($form['actions']['submit'])) {
+    $submit_classes = ['me-md-3', 'btn', 'btn-primary', 'btn-md'];
+    foreach ($submit_classes as $submit_class) {
+      if (!isset($submit_class, $form['actions']['submit']['#attributes']['class']) || !in_array($submit_class, $form['actions']['submit']['#attributes']['class'], TRUE)) {
+        $form['actions']['submit']['#attributes']['class'][] = $submit_class;
+      }
+    }
+  }
+  if (isset($form['actions']['reset'])) {
+    $reset_classes = ['btn', 'btn-light', 'btn-md'];
+    foreach ($reset_classes as $reset_class) {
+      if (!isset($form['actions']['reset']['#attributes']['class']) || !in_array($reset_class, $form['actions']['reset']['#attributes']['class'], TRUE)) {
+        $form['actions']['reset']['#attributes']['class'][] = $reset_class;
+      }
+    }
+  }
+}
+
+/**
+ * Helper function to preprocess the facets_summary item list.
+ *
+ * @param array $variables
+ *   Render array variables.
+ */
+function oe_whitelabel_search_preprocess_facets_summary_item_list(array &$variables) {
+  $variables['result_items'] = [];
+  foreach ($variables['items'] as $item) {
+    if ($item['value']['#type'] === 'link') {
+      $item['value']['#attributes']['class'] = 'standalone';
+      $variables['result_items'][] = $item;
```

Comment:
this is not a menu, we don't need to have a separate key to avoid conflicts

Reply:
We need because the "result count" with the number of the total is also an item.